digitalpublicgoods/mochcare.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "MOCHCare",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.mochcare.com"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://mochcare.gitbook.io/product-docs/59ojkfJv1gOmOLD9PiZD/",
+      "https://github.com/MochCare-Gh/service-provider-portal"
```

Comment:
This link shows incomplete documentation.It's currently not possible for a technical person unfamiliar with the project to launch and run the software without making lots of assumptions. This could be a pain-point in the future for any possible codebase improvements or refactors. Thanks.